[ DEBUG ][ 2022-07-03 10:55:52,681 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>
[ DEBUG ][ 2022-07-03 10:55:53,055 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>
[ DEBUG ][ 2022-07-03 10:55:53,059 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>
[ DEBUG ][ 2022-07-03 10:55:53,071 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>
[ DEBUG ][ 2022-07-03 10:55:53,078 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>
[ DEBUG ][ 2022-07-03 10:55:53,094 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>
[ DEBUG ][ 2022-07-03 10:55:53,099 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>
[ DEBUG ][ 2022-07-03 10:55:53,103 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>
[ DEBUG ][ 2022-07-03 10:55:53,106 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>
[ DEBUG ][ 2022-07-03 10:55:56,420 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>
[ DEBUG ][ 2022-07-03 10:55:56,421 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>
[ DEBUG ][ 2022-07-03 10:55:56,421 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>
[ DEBUG ][ 2022-07-03 10:55:56,422 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>
[ DEBUG ][ 2022-07-03 10:55:56,423 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>
[ DEBUG ][ 2022-07-03 10:55:56,424 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>
[ DEBUG ][ 2022-07-03 10:55:56,425 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>
[ DEBUG ][ 2022-07-03 10:55:56,426 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>
[ DEBUG ][ 2022-07-03 10:55:56,427 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>
[ DEBUG ][ 2022-07-03 11:27:10,216 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>
[ DEBUG ][ 2022-07-03 11:27:10,361 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>
[ DEBUG ][ 2022-07-03 11:27:10,363 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>
[ DEBUG ][ 2022-07-03 11:27:10,365 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>
[ DEBUG ][ 2022-07-03 11:27:10,367 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>
[ DEBUG ][ 2022-07-03 11:27:10,371 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>
[ DEBUG ][ 2022-07-03 11:27:10,373 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>
[ DEBUG ][ 2022-07-03 11:27:10,376 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>
[ DEBUG ][ 2022-07-03 11:27:10,378 ][ base ] ==> Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/core/handlers/base.py", line 100, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/home/ymane/.local/lib/python3.8/site-packages/django/urls/resolvers.py", line 575, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<URLResolver <URLPattern list> (admin:admin) 'admin-back-office/'>], [<URLResolver <module 'django.conf.urls.i18n' from '/home/ymane/.local/lib/python3.8/site-packages/django/conf/urls/i18n.py'> (None:None) 'i18n/'>], [<URLResolver <module 'session_security.urls' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/session_security/urls.py'> (None:None) 'session_security/'>], [<URLResolver <module 'oauth.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/oauth/urls/base.py'> (None:None) 'oauth/'>], [<URLResolver <module 'sentry.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/sentry/urls/base.py'> (None:None) 'sentry/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '' [name='frontspace']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-lieu_depart/' [name='ajax_load_lieu_depart']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern 'ajax/load-destination/' [name='ajax_load_destination']>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.frontspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/frontspace.py'> (None:None) '^'>, <URLPattern '^static/(?P<path>.*)$'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.waiting_page' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/waiting_page.py'> (None:None) 'wait/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.payement_state' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/payement_state.py'> (None:None) 'payement_state/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLResolver <module 'website.urls.userspace' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/userspace.py'> (None:None) 'back-office/'>], [<URLResolver <module 'website.urls.base' from '/home/ymane/Téléchargements/projet_complet_easytrans_et_git/EASYTRANS/website/urls/base.py'> (None:None) ''>, <URLPattern '^media/(?P<path>.*)$'>], [<URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>], [<URLPattern '^media/(?P<path>.*)$'>], [<URLPattern '^static/(?P<path>.*)$'>]], 'path': 'website/front/js/main.js'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 828, in _resolve_lookup
    current = current[bit]
TypeError: 'URLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 836, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'URLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 842, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ymane/.local/lib/python3.8/site-packages/django/template/base.py", line 847, in _resolve_lookup
    raise VariableDoesNotExist("Failed lookup for key "
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in <URLResolver <URLPattern list> (ninja:api-1.0.0) 'api-ninja/'>
